### PR TITLE
10 BLB/BLC cards, batch 2

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/hazel_of_the_rootbloom.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Hazel of the Rootbloom
+ManaCost:2 B G
+Types:Legendary Creature Squirrel Druid
+PT:3/5
+A:AB$ Mana | Cost$ T PayLife<2> tapXType<X/Token> | Produced$ Combo Any | Amount$ X | SpellDescription$ Add X mana in any combination of colors.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ DBCopyPermanent | Secondary$ True | TriggerDescription$ At the beginning of your end step, create a token that's a copy of target token you control. If that token is a Squirrel, instead create two tokens that are copies of it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hazel_of_the_rootbloom.txt:6]: why secondary?

File: forge-gui/res/cardsfolder/upcoming/hunters_talent.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+SVar:TrigHit:DB$ Pump | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ SoulsDamage | StackDescription$ None
+SVar:SoulsDamage:DB$ DealDamage | ValidTgts$ Creature.YouDontCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you don't control | NumDmg$ X | DamageSource$ ParentTarget
+K:Class:2:1 G:AddTrigger$ TriggerAttackersDeclared
+SVar:TriggerAttackersDeclared:Mode$ AttackersDeclared | AttackingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPump | Secondary$ True | TriggerDescription$ Whenever you attack, target attacking creature gets +1/+0 and gains trample until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature | NumAtt$ 1 | KW$ Trample
+K:Class:3:3 G:AddTrigger$ TriggerEndStep
+SVar:TriggerEndStep:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | IsPresent$ Creature.powerGE4 | Execute$ TrigDraw | TriggerDescription$ At the beginning of your end step, if you control a creature with power 4 or greater, draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hunters_talent.txt:11]: ispresent incomplete

File: forge-gui/res/cardsfolder/upcoming/kastral_the_windcrested.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Kastral, the Windcrested
+ManaCost:3 W U
+Types:Legendary Creature Bird Scout
+PT:4/5
+K:Flying
+T:Mode$ DamageDoneOnce | CombatDamage$ True | ValidSource$ Creature.Bird+YouCtrl | ValidTarget$ Player | TriggerZones$ Battlefield | Execute$ TrigCharm | TriggerDescription$ Whenever one or more Birds you control deal combat damage to a player, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBChangeZone,DBPutCounterAll,DBDraw
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Hand,Graveyard | Destination$ Battlefield | ChangeType$ Creature.Bird+YouOwn | WithCountersType$ FINALITY | Optional$ You | SpellDescription$ You may put a Bird creature card from your hand or graveyard onto the battlefield with a finality counter on it.
+SVar:DBPutCounterAll:DB$ PutCounterAll | ValidCards$ Bird.YouCtrl | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on each Bird you control.
+SVar:DBDraw:DB$ Draw
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kastral_the_windcrested.txt:10]: description missing

**Review by tool4ever — APPROVED** [review]
nice work

File: forge-gui/res/cardsfolder/upcoming/hazel_of_the_rootbloom.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Hazel of the Rootbloom
+ManaCost:2 B G
+Types:Legendary Creature Squirrel Druid
+PT:3/5
+A:AB$ Mana | Cost$ T PayLife<2> tapXType<X/Token> | Produced$ Combo Any | Amount$ X | SpellDescription$ Add X mana in any combination of colors.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ DBCopyPermanent | TriggerDescription$ At the beginning of your end step, create a token that's a copy of target token you control. If that token is a Squirrel, instead create two tokens that are copies of it.
+SVar:CopyArtifact:DB$ CopyPermanent | ValidTgts$ Permanent.token+YouCtrl | NumCopies$ Y | TgtPrompt$ Select target token you control to copy
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/hazel_of_the_rootbloom.txt:7]: @tool4ever @Fulgur14 Wrong Svar: "DBCopyPermanent" vs "CopyArtifact"

**Pull request merged**